nurseconnect/tests/test_forms.py

```diff
@@ -2,66 +2,99 @@
 from django.test import TestCase
 
 from molo.core.tests.base import MoloTestCaseMixin
-from molo.profiles import models
+# from molo.profiles.models import SecurityQuestion
 
 from nurseconnect import forms
 
 
 class RegisterFormTestCase(MoloTestCaseMixin, TestCase):
     def setUp(self):
-        self.form = forms.RegistrationForm
```

Comment:
I think it could be best to separate each of the forms to its own `TestCase`. What do you think?